sample-app/src/main/java/com/robustwealth/microservices/sample/repository/common/SqlReader.java

```diff
@@ -37,6 +37,6 @@ public String getSql(String queryName) {
 
     @Nonnull
     private String getBaseResourceFromRepositoryClassName() {
-        return repositoryClass.getName().replaceAll("\\.", File.separator);
+            return repositoryClass.getName().replaceAll("\\.", File.separator + File.separator);
```

Comment:
I do not believe that this will be a workable solution across platforms.  This may work on Windows (because the file separator is a backslash, but on Mac/Linux, it is a forward slash.  Additionally, reading a resource via Java should be the same across platforms.

Reply:
This is the code that had the two file separators.  Why is this change needed?

Reply:
This is the outdated version. The latest commit reverted the double file separator

Reply:
I see.  Thanks for clarifying.